data_taking_scripts/motor_control.py

```diff
@@ -54,17 +54,18 @@ def inch_to_cm(dist):
         #log narrowscan
         logger.log_modemap(narrow_scan_start_freq, narrow_scan_stop_freq, sec_wait_for_na_averaging, 'narrowscan')
 
-        mirror_spacing_tracker, new_plate_separation = orpheus_motors.move_by_increment(increment_distance,
-                                                                                   plate_thickness,
-                                                                                   mirror_spacing_tracker,
-                                                                                   num_plates,
-                                                                                   initial_plate_separation)
-        orpheus_motors.wait_for_motors()
         i = round((i+inch_to_cm(increment_distance)),4)
```

Comment:
@parashar2399 I recommend changing this variable to something like `change_in_resonator_length` or `delta_length`. The i usually means an incrementing variable, like i = 1, 2,3. So this confused me.